### PR TITLE
Adds Charles Barkley and Riblin joke abnormalities

### PR DESCRIPTION
**Pull request by vampirebat74**
## About The Pull Request

What it does on the tin

## Why It's Good For The Game

Should make some things more accessible

## Changelog
:cl:
add: Adds Charles Barkley and the Riblin
imageadd: Adds relevant abnormality portraits
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		icon_state = icon_living
+
+/mob/living/simple_animal/hostile/abnormality/barkley/proc/ChaosDunk()
+	show_global_blurb(50, "Chaos Dunk")
+	Explode()
+	Cinematic(CINEMATIC_CHAOS_DUNK, world)
+
+/mob/living/simple_animal/hostile/abnormality/barkley/proc/Explode()
+	set waitfor = FALSE
+	sleep(50)
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm:65]: its clearer how much it waits if you put the SECONDS define there
```suggestion
	sleep(5 SECONDS)
```

File: code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	)
+	abnormality_origin = ABNORMALITY_ORIGIN_ORIGINAL
+
+	var/explosion_amt = 3
+
+/mob/living/simple_animal/hostile/abnormality/barkley/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	if(prob(30))
+		datum_reference.qliphoth_change(-1)
+	return
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm:39]: don't put returns at the end of code, it automatically puts them there
```suggestion

```

File: code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+/mob/living/simple_animal/hostile/abnormality/barkley/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	datum_reference.qliphoth_change(-1)
+	return
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm:44]: don't put returns at the end of code, it automatically puts them there
```suggestion

```

File: code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+/mob/living/simple_animal/hostile/abnormality/barkley/ZeroQliphoth(mob/living/carbon/human/user)
+	. = ..()
+	ChaosDunk(user)
+	return
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/aleph/barkley.dm:49]: don't put returns at the end of code, it automatically puts them there
```suggestion

```

File: code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	)
+	abnormality_origin = ABNORMALITY_ORIGIN_ORIGINAL
+
+/mob/living/simple_animal/hostile/abnormality/wellcheers/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	var/obj/item/serving = /obj/item/food/mcrib
+	var/turf/dispense_turf = get_step(src, pick(NORTH, SOUTH, NORTHEAST, NORTHWEST, SOUTHEAST, SOUTHWEST))
+	dropped_can = new serving(dispense_turf)
+	visible_message(span_notice("[src] offers a [dropped_can]."))
+	return
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm:36]: don't put returns at the end of code, it automatically puts them there
```suggestion

```

File: code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+// Death!
+/mob/living/simple_animal/hostile/abnormality/wellcheers/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	user.gib()
+	return
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm:42]: don't put returns at the end of code, it automatically puts them there
```suggestion

```

**Review by Gboster-0 — COMMENTED** [review]
those returns are not good

File: code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm
Comment (Coxswain-Navigator) [inline code/modules/mob/living/simple_animal/abnormality/zayin/riblin.dm:1]: This is a joke abnormality so remove its ability to spawn.

File: code/modules/projectiles/guns/ego_gun/joke.dm
Comment (Coxswain-Navigator) [inline code/modules/projectiles/guns/ego_gun/joke.dm:1]: Change the firing and/or projectile hit sound so it's less annoying and immersion breaking.

**Review by Coxswain-Navigator — CHANGES_REQUESTED** [review]
April fools is now over and this now needs a somewhat serious review. Here are some changes I would like made.

**Review by Coxswain-Navigator — APPROVED** [review]
This was made to my specifications, and I approve.

**Review by Kitsunemitsu — APPROVED** [review]
PR Approved, they're all joke abnos.

**Pull request merged**